packages/homepage/package.json

```diff
@@ -37,8 +37,7 @@
   "license": "MIT",
   "main": "n/a",
   "scripts": {
-    "patch": "sed -i -e 's/if(c\\.initial) return;/if(!c||c.initial) return;/g' ../../node_modules/extract-text-webpack-plugin/index.js",
```

Comment:
Hmm, maybe we need this one still.

Reply:
From what I've found, this was fixing an old `extract-text-webpack-plugin` issue which got fixed since.

Reply:
Okay nice! In that case let's remove it!